docs/modules/ROOT/pages/api/access.adoc

```diff
@@ -48,6 +61,18 @@ This module includes the internal `assert_only_owner` to restrict a function to
 * xref:OwnableComponent-renounceOwnership[`++renounceOwnership(self)++`]
 --
 
+[.contract-index]
+.Embeddable Implementations (camelCase two-step transfer)
```

Comment:
Removing these two lines doesn't actually do anything to how the page is rendered.

Though if I understand your meaning correctly, you want to have just a single "Embeddable Implementations (CamelCase)" section? If so, how would folks differentiate between one-step and two-step API? Should I then merge the non-camel sections just above as well?

Reply:
they differentiate between APIs because functions are grouped below different impls

Reply:
camel and non camel should be merged, yes

Reply:
By merging camel and non camel (they weren't before however) we loose the differentiation between one step and two step impls - how to tell in which `transfer_ownership` is a reader interested in? Which is related to the other unresolved convo below.

How about having 2 sections - "Embeddable Implementations" and "Embeddable Implementations (two step)" where camel and non camel functions are merged and then lower in Embeddable functions section (see img just to be clear) have a similar structure for one step and two step API? 

<img width="720" alt="Screenshot 2023-12-02 at 22 14 37" src="https://github.com/OpenZeppelin/cairo-contracts/assets/184055/10a31df5-fb73-48cd-9590-8998da7b849e">

Reply:
what about having one section per impl? not sure we need the "embeddable" section, it might be enough with the distinction at the component level, then function docs can be grouped just by impl

Reply:
this is what I was suggesting btw
![image](https://github.com/OpenZeppelin/cairo-contracts/assets/1703831/c61d672e-412c-41ef-81c7-6ce3edb9d2d0)

Reply:
I've done the grouping as suggested above. Should I also move around the function details themselves? Because now they are in different order than in this overview table.

Reply:
let's order them properly if possible! thanks